src/client/components/equalizer/streamButton/streamButton.css

```diff
@@ -0,0 +1,12 @@
+.StreamButton{
+  font-family: var(--font-ordinary);
+  font-size: var(--fontSize-text);
+  color: var(--mainText-ordinary-white);
+  border: 1px solid var(--mainText-accent-blue);
+  background-color: var(--rangeSlider-ocean);
+
+  width: 200px;
+  height: 30px;
+  letter-spacing: 1px;
+  border-radius: 4px;
+}
```

Comment:
missed empty line at the end of the file.

Reply:
done)